packages/logger/README.md

```diff
@@ -41,10 +41,58 @@ npm install @sapphire/plugin-logger @sapphire/framework
 
 ## Usage
 
+This registers the necessary options and methods in the Sapphire client to be able to use the log plugin.
+
 ```typescript
+// Main bot file
 import '@sapphire/plugin-logger/register';
 ```
 
+To be able to use the Logger anywhere you first need to import the `containter` property from the `@Sapphire/framework` package.
+
+```typescript
+import { container } '@sapphire/plugin-logger';
+
+export class MyAwesomeService {
+	public printAwesomeLog() {
+		container.logger.log('log message');
+	}
+}
+```
+
+In pieces (commands, arguments, preconditions, etc.), the container is already available on the class as `this.container`. Therefore there is often no need to specifically import the container in such files to access the Logger methods.
+
+```typescript
+// ping command
+
+import { Command, CommandOptions, PieceContext } from '@sapphire/framework';
+import type { Message } from 'discord.js';
+
+export class PingCommand extends Command {
+	public constructor(context: PieceContext, options: CommandOptions) {
+		super(context, {
+			...options,
+			description: 'ping pong'
+		});
+	}
+
+	public async run(message: Message) {
+		this.container.logger.warn('warning message');
+	}
+}
+```
+
+## Types of logs
+
+1. trace
+2. debug
+3. info
+4. warn
+5. error
+6. fatal
+
+> Example: `container.logger.debug("log debug message");`
```

Comment:
Maybe make this use singlequotes for consistency?

Reply:
done. 📮